reviews/linux-kernel.md

```diff
@@ -0,0 +1,69 @@
+## Front Metadata
```

Comment:
Sorry, the template isn't clear on this, but the review needs to start with this text. I added both OSTIF and Atredis as reviewers, and made an assumption that neither are directly associated with the kernel project (full disclosure / bias), and that any compensation received from doing the review wasn't coming directly from the kernel project.

```
---
Publication-State: Active
Access: Public
Reviewers:
- Organization: Open Source Technology Improvement Fund
  Associated-With-Project: false
  Compensation-Source: External
- Organization: Atredis Partners
  Associated-With-Project: false
  Compensation-Source: External  
Domain: Security
Methodology:
- External-Review
Issues-Identified: Non-Severe
Package-URLs:
- pkg:generic/linux-kernel?download_url=https://kernel.org
- pkg:github/torvalds/linux
Review-Date: 2021-01-15
Scope: Non-Implementation
Schema-Version: 1.0
SPDX-License-Identifier: CC-BY-4.0
---
```

Reply:
Thank you @scovetta for making those updates. Your assumptions are indeed correct. The review was sponsored by Linux Foundation. Sorry if the format is wonky as well, I thought to add a folder "linux kernel" similar to npm and zlib but wasn't sure how. Please let me know if there's anything I can do. Thanks again!